File: DDIT_Python/Hello_Python/day07/myselect.py
import pymysql

# pymysql.connect(host=서버IP주소, user=사용자, passoword=암호, db=데이터베이스, charset=문자세트)
conn = pymysql.connect(host="localhost", user="root", password="python", db="python", charset="utf8", port=3305)
cursor = conn.cursor(pymysql.cursors.DictCursor) 

sql = "select * from emp" 

cursor.execute(sql) 
# cursor: 자바의 statement

rows = cursor.fetchall()

print(rows)
    
cursor.close()
conn.close() 

File: DDIT_Python/Hello_Node/myselect.js
var mysql = require('mysql');
var connection = mysql.createConnection({
	host: 'localhost',
	user: 'root',
	password: 'python',
	database: 'python',
	port: 3305
});

connection.connect();

connection.query('SELECT * from emp', function (error, results, fields) {

	console.log("-----------------------------------");
	console.log(error);
	console.log("-----------------------------------");
	console.log(results);
	console.log("-----------------------------------");
	console.log(fields);
});

connection.end();
